R/linear_reg.R

```diff
@@ -1,71 +1,49 @@
-#' General Interface for Linear Regression Models
+#' Linear regression
+#'
+#' @description
+#'
+#' `linear_reg()` defines a model that can predict numeric values from
+#' predictors using a linear function.
+#'
+#' There are different ways to fit this model. Information about the available
+#' _engines_ that can be used for fitting at:
+#'
+#' \Sexpr[stage=render,results=rd]{parsnip:::find_engine_files("linear_reg")}
+#'
+#' Loading model `parsnip`-adjacent packages will show additional engines for
+#' this model. The `multilevelmod` package contains additional engines for
+#' `linear_reg()`.
```

Comment:
```suggestion
#' Loading `parsnip`-adjacent packages which contain more engines for 
#' this model will make the additional engines show up here. For example,
#' the [multilevelmod](https://github.com/tidymodels/multilevelmod/) package 
#' contains additional engines for `linear_reg()`.
```
Do we want to point to multilevelmod when it's not on CRAN yet?

Reply:
I'll take that sentence out until it is ready.